erpnext/projects/doctype/project/project.py

```diff
@@ -836,7 +836,7 @@ def validate_depreciation(self):
 			item_codes_visited.add(d.depreciation_item_code)
 
 	def validate_warranty(self):
```

Comment:
Lets add a `def validate_feedback` or some method that does this
`if self.get('customer_feedback') and not self.get('feedback_date'): set date/time`